File: episode-42-prolog/fizzbuzz.pl
divisible(N, D) :- 0 is N mod D.
not_divisible(N, D) :- \+ divisible(N, D).

fizzbuzz(N, 'FizzBuzz') :- divisible(N, 15).
fizzbuzz(N, 'Fizz') :- divisible(N, 3), not_divisible(N, 5).
fizzbuzz(N, 'Buzz') :- divisible(N, 5), not_divisible(N, 3).
fizzbuzz(N, N) :- not_divisible(N, 3), not_divisible(N, 5).

num_between(X, A, B) :- A =< B, X = A.
num_between(X, A, B) :- A < B, A1 is A+1, num_between(X, A1, B).

write_fizzbuzz(X) :- fizzbuzz(X, W), write(W), nl.

fizzbuzzloop :-
  num_between(X, 1, 100),
  write_fizzbuzz(X),
  fail.

File: episode-56-php/fizzbuzz.php
<?php
for ($n = 1; $n <= 100; $n += 1) {
  if ($n % 15 == 0) {
    echo "FizzBuzz\n";
  } else if ($n % 5 == 0) {
    echo "Buzz\n";
  } else if ($n % 3 == 0) {
    echo "Fizz\n";
  } else {
    echo "$n\n";
  }
}
?>
